lib/Raven/Configuration.php

```diff
@@ -0,0 +1,446 @@
+<?php
+
+/*
+ * This file is part of Raven.
+ *
+ * (c) Sentry Team
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Raven;
+
+use Composer\CaBundle\CaBundle;
+use Symfony\Component\OptionsResolver\Options;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * Configuration container for the Sentry client.
+ *
+ * @author Stefano Arlandini <sarlandini@alice.it>
+ */
+class Configuration
```

Comment:
Because

- The ruby SDK has it, so we are matching that
- Having a separate class for the options means that we don't have to have a monolithic class (the `Client` class) that stores evyerthing
- Processors (and probably a lot of new features) require or will require access to the configuration of the client. Passing a neutral class instead of the `Client` instance means that there won't be anymore a circular reference between them and the client itself
- It's common in other libraries to have such a class to store the configuration of a client